include/Git.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+#include "macros.h"
+
+class Git {
+public:
+    static std::string getCurrentBranch();
```

Comment:
If a class has just one static function, maybe it shouldn't be a class. It should probably just be a function